packages/web/src/pages/audio-rewards-page/ChallengeRewardsTile.tsx

```diff
@@ -164,12 +164,37 @@ const useRewardIds = (
   return filteredRewards
 }
 
+const sortChallenges = (
+  userChallenges: Record<string, UserChallenge>
+): ((id1: ChallengeRewardID, id2: ChallengeRewardID) => number) => {
+  return (id1, id2) => {
+    if (Object.keys(userChallenges).length === 0) {
```

Comment:
Is this for an empty object check? Any reason the other use doesn't have this check?

Also, for simplicity, maybe this could be checks like `if (!userChallenge1)` in the lines below?

Reply:
Yep empty object, your suggestion makes sense. And good call I should probably add it to the other one as well.

Reply:
if both are the same, we can probably keep it in utils then right?